src/UI/Areas/Identity/Pages/Account/Manage/TwoFactorAuthentication.cshtml

```diff
@@ -29,6 +29,12 @@
         </div>
     }
 
+    if (Model.IsMachineRemembered)
+    {
+        <form method="post" style="display: inline-block">
+            <button type="submit" class="btn btn-default">Forget machine</button>
```

Comment:
Should this be `Forget browser`? Maybe a PM could help here.

Reply:
@blowdart text help?

Reply:
Maybe? "Forget two-factor for this machine"

Reply:
Forget this browser